spec/controllers/teams_controller_spec.rb

```diff
@@ -71,13 +46,6 @@
         expect(team.team_members.first.current_member?).to be false
       end
 
-      it 'cannot demote oneself' do
```

Comment:
This feature is still available in banned competitors, but not used now. Also cannot create an alternate test case because banned people cannot even go to the edit page.